content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15.md

```diff
@@ -314,8 +314,9 @@ without compromising the minimum required capacity for running your workloads.
     {{< tabs name="k8s_kubelet_and_kubectl" >}}
     {{% tab name="Ubuntu, Debian or HypriotOS" %}}
     # replace x in 1.15.x-00 with the latest patch version
-    apt-get update
-    apt-get install -y kubelet=1.15.x-00 kubectl=1.15.x-00
+    apt-mark unhold kubelet kubectl && \
```

Comment:
If a cluster operator didn't have a hold set, this will put one in place, right?

Is that worth calling out? (I think it might be).

Reply:
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#installing-kubeadm-kubelet-and-kubectl

Follow the instructions in the install documentation to do apt-mark hold.
When I tried upgrade, I was worried about this part.
I think that the procedure is a level that can be solved on the spot, but I thought it would be better to have consistency as a document.